packages/editor/src/components/parts/common/table/useResizableEditableTable.ts

```diff
@@ -31,20 +38,24 @@ const useResizableEditableTable = <TData>({ data, columns, onChange, emptyDataOb
     getSortedRowModel: getSortedRowModel(),
     meta: {
       updateData: (rowId, columnId, value) => {
-        const rowIndex = parseInt(rowId);
-        const updatedData = data.map((row, index) => {
-          if (index === rowIndex) {
-            return {
-              ...data[rowIndex]!,
-              [columnId]: value
-            };
-          }
-          return row;
-        });
-        if (!deepEqual(updatedData[updatedData.length - 1], emptyDataObject) && rowIndex === data.length - 1) {
-          onChange([...updatedData, emptyDataObject]);
+        if (specialUpdateData) {
+          specialUpdateData(rowId, columnId, value);
         } else {
-          onChange(updatedData);
+          const rowIndex = parseInt(rowId);
+          const updatedData = data.map((row, index) => {
+            if (index === rowIndex) {
+              return {
+                ...data[rowIndex]!,
+                [columnId]: value
+              };
+            }
+            return row;
+          });
+          if (!deepEqual(updatedData[updatedData.length - 1], emptyDataObject) && rowIndex === data.length - 1) {
+            onChange([...updatedData, emptyDataObject]);
+          } else {
+            onChange(updatedData);
+          }
```

Comment:
instead of blow this up, I would create a updateData function outside of this meta object and override it with the specialUpdateData stuff if available... Or maybe even more can be shared between those two?

Reply:
do you mean something like this?
![grafik](https://github.com/axonivy/inscription-client/assets/141223521/f94c5e04-cd0d-4f4a-96c8-74047f08b963)

Reply:
Yes I had something like this in mind